src/niscope/system_tests/test_system_niscope.py

```diff
@@ -457,18 +520,29 @@ def session_creation_kwargs(self):
 
     # not supported by grpc due to numpy usage
     @pytest.mark.parametrize(
-        "fetch_waveform_type,type_min_value",
+        "fetch_waveform_type,type_min_value,test_channels,test_channels_expanded",
         [
-            (numpy.int8, numpy.iinfo(numpy.int8).min),
-            (numpy.int16, numpy.iinfo(numpy.int16).min),
-            (numpy.int32, numpy.iinfo(numpy.int32).min),
-            (numpy.float64, numpy.finfo(numpy.float64).min),
+            (numpy.int8, numpy.iinfo(numpy.int8).min, test_channels_1, test_channels_1_expanded),
```

Comment:
Given that the bug is in `fetch_into` and not the private methods that handle calls for specific types, perhaps there's no need to duplicate the numpy parameterization for each test_channels value?

Reply:
Would that make the test effective only when tied to the current implementation? If so, better leave it parameterized.